web/packages/teleterm/package.json

```diff
@@ -45,7 +45,7 @@
     "electron": "31.0.1",
     "electron-notarize": "^1.2.2",
     "electron-vite": "^2.2.0",
-    "immer": "^9.0.7",
+    "immer": "^10.1.1",
```

Comment:
@gzdunek Looking at v10 release notes, I don't think anything affects us. The only thing I'm unsure about are the getters and setters. It seems it only applies when creating custom "immerable" classes (search for `useStrictShallowCopy` on https://immerjs.github.io/immer/complex-objects/) which we don't do anywhere.

Reply:
Yeah, I agree, we should be safe to upgrade.